### DISCUSSION TITLE
WEEK 8 Accessing Data Issue

### DISCUSSION THREAD

**Discussion by taylor-obrien** [earthlab-education/Earth-Analytics-2023-01-Intro#550] [Discussion submissions]
I am trying to request access for data through the M2M interface. In the video, Elsa says to just put that we are with the geography department. However, requesting access requires a lot of information and I'm not sure what to put. It includes: datasets required, data products required, scenes required, and scripting capabilities. Does anyone know how I should fill this out?
Thank you!

**Comment by annettebalmes** [comment]
Here is what I put to get my approval, it took about a day and a half to get approved. I can't see the form anymore since I was already approved but here's what I can remember:

* Access to EE's Machine to Machine interface (MACHINE)
* Datasets required: NAIP (There was a question about description above so I put NAIP Mulitspectral data there)
* Data products required: I just put Dataset download
* Scenes Required: I think I put 50 (yearly), if you try running the code without access yet it tells you how many scenes it thinks it needs for all of the neighborhoods
* Scripting capabilities I put python earthpy.earthexplorer
* Last desciption: I copied and pasted from the assignment: 
1. Download National Agricultural Imagery Program (NAIP) multispectral data for a single neighborhood in Chicago
2. Plot True Color (RGB) and Color Infrared (CIR) images of the area
3. Calculate summary statistics of the NDVI.
4. Then do the same for all neighborhoods (Added this myself)

**Comment by ellenalamont17** [comment]
I also kept mine really simple:

* Access to EE's Machine to Machine interface (MACHINE)
* Datasets required: NAIP 
* Data products required: NAIP
Scenes Required: I think I put something like 5-10/month.
Scripting capabilities python
Last desciption: I stated I was a student at UC Boulder and that I would be using the data for educational purposes learning how to access, download, manipulate, and analyze the data.

I got my approval in less an 24 hours.
